table/block_based_table_reader.cc

```diff
@@ -2082,14 +2086,18 @@ void BlockBasedTableIterator::FindKeyBackward() {
       return;
     }
 
-    auto ukey = user_key();
-    if (icomp_.user_comparator()->Compare(ukey, *range_tombstone_.start_key()) <
+    ParsedInternalKey parsed;
```

Comment:
Huh, seems kind of weird that there's no already-parsed internal key around.

Reply:
I couldn't find one. This is one of the fundamental changes I made in pebble: pass around a decomposed internal key type everywhere.